src/modules/shared/components/CopyButton/CopyButton.tsx

```diff
@@ -7,45 +7,61 @@ import { toastService } from '@shared/services/toast-service';
 
 import { Icon, IconNamesLight } from '../Icon';
 
-const CopyButton: FC<ButtonProps & { text: string; enableToast?: boolean }> = (props) => {
-	const { tHtml, tText } = useTranslation();
-	const { onClick, text, enableToast = true } = props;
-
-	const clickHandler = useCallback(
-		(e: MouseEvent<HTMLButtonElement>) => {
-			e.preventDefault();
-			e.stopPropagation();
-
-			const copied = copy(text);
-
-			if (copied && enableToast) {
-				toastService.notify({
-					title: tHtml('modules/shared/components/copy-button/copy-button___success'),
-					description: tHtml(
-						'modules/shared/components/copy-button/copy-button___text-was-copied-to-your-clipboard',
-						{
-							text,
-						}
-					),
-				});
-			}
-
-			onClick?.(e);
-		},
-		[onClick, text, enableToast, tHtml]
-	);
-
-	return (
-		<Button
-			icon={<Icon name={IconNamesLight.Copy} aria-hidden />}
-			aria-label={tText(
-				'modules/shared/components/copy-button/copy-button___kopieer-naar-klembord'
-			)}
-			variants="sm"
-			{...props}
-			onClick={clickHandler}
-		/>
-	);
-};
+const CopyButton: FC<ButtonProps & { text: string; enableToast?: boolean; isInputCopy?: boolean }> =
```

Comment:
Instead of using the `isInputCopy` boolean to create different buttons, I'd pass the values as properties. This will make this component more flexible and more future proof. 

Since the `CopyButton` has the same props as the `Button` component, you can pass the label, icons, variants,... 

e.g. https://codesandbox.io/s/aged-http-ke5y7p?file=/src/App.js